runtime/src/main/java/org/corfudb/runtime/view/replication/QuorumReplicationProtocol.java

```diff
@@ -131,6 +132,10 @@ public void write(Layout layout, ILogData data) throws OverwriteException {
         }
     }
 
+    /** {@inheritDoc} */
+    @Override
+    public void writeForwardpointer(Layout layout, UUID stream, long address, long nextAddress) {
```

Comment:
any thoughts on this? should we care that only a quorum might have the actual entry?